c/contracts.h

```diff
@@ -5,12 +5,19 @@
 #include "ripemd160.h"
 #include "sha256.h"
 #include <intx/intx.hpp>
-#include <bn128.hpp>
 
 #include "polyjuice_utils.h"
 #include "sudt_contracts.h"
 #include "other_contracts.h"
 
+/**
+ * alt_bn128_staticlib
+ * @see https://github.com/mohanson/bn/blob/d1c73cc198a14ca894a37ba99df88a9fb812a43c/alt_bn128_staticlib/src/ethereum.rs
+ */
+extern "C" uint32_t alt_bn128_add(const uint8_t *data, uint32_t data_len, uint8_t *output);
```

Comment:
It's better to return "uint64_t" or "int" as the error code.

Reply:
The extern "C" functions use `u32` as a return code:
https://github.com/mohanson/bn/blob/master/alt_bn128_staticlib/src/ethereum.rs#L2

should I use int in c/contracts.h?

Reply:
it seems `u32` is OK here.